pattern-library/sass/patterns/_buttons.scss

```diff
@@ -120,8 +120,8 @@ $btn-small-font-size: font-size(small);
     &.is-hovered,
     &:focus,
     &.is-focused {
-        border-color: palette(primary, accent);
-        background: palette(primary, accent);
+        border-color: $btn-focus-color;
```

Comment:
Thanks for making these into variables! We should strive never to use ``palette`` directly in our Sass, but instead use variables. This allows a theme to override the value for the variables and get a consistent experience.